test/android/core/javatests/src/org/xwalk/core/xwview/test/AddandRemoveJavascriptInterfaceTest.java

```diff
@@ -53,17 +63,25 @@ private void raisesExceptionAndSetTitle(String script) throws Throwable {
     }
 
     @SmallTest
-    @Feature({"AddJavascriptInterface"})
-    public void testAddJavascriptInterface() throws Throwable {
+    @Feature({"AddandRemoveJavascriptInterface"})
+    public void testAddandRemoveJavascriptInterface() throws Throwable {
         final String name = "add_js_interface.html";
 
         addJavascriptInterface();
         loadAssetFile(name);
         assertEquals(mExpectedStr, getTitleOnUiThread());
+
+        removeJavascriptInterface();
```

Comment:
These tests can definitely be more comprehensive. Looking at this method, I don't know if:
- Removing an interface without reloading the page really does not have any effect.
- Removing an interface that has not been added does not cause any error.
- Adding more than 1 interface and removing one of them keeps all the others intact.

When writing tests, you really have to think of corner cases and weird possibilities, not just verify that the standard behavior works.